src/components/screens/gallery.js

```diff
@@ -1,11 +1,16 @@
 import styles from '../styles/gallery.module.css'
-import NavBar from '../features/navBar.js'
+import Header from '../features/header'
+import Footer from '../features/footer'
 
 function Gallery({events}) {
     return(
-        <div className={styles.main}>
-          <NavBar/>
+        <div className={styles.header} >
+            <Header/>
+                <div className={styles.footer} >
+                    <Footer/>
+                </div>
```

Comment:
There is a footer div style inside of the header div style here, maybe try cleaning up to have the main style called 'main' or 'gallery' & then you won't need divs for your abstracted divs, the style for your header and footer will solely live in the footer.module.css this way